### ISSUE TITLE
Enemies Generate Inside Exit Portal

### ISSUE THREAD

**Issue by yhbautista** [dmr-capstone/dmr-capstone#171]
In certain dungeon generations, enemies will generate inside of the exit portal. Due to issue #170, this makes it impossible to actually hit the enemy before stepping into the portal.

**Comment by pagewin** [comment]
Not sure if really a bug to fix

1. Can prevent enemies from spawning on same tile as exit portal, but this doesn't stop them from walking into it naturally
2. Solving #170 will make it so this isn't a big deal

**Comment by yhbautista** [comment]
If the group majorly thinks this is an unnecessary issue, ya'll can feel free to mark it as "won't fix" and close it. I just thought I would report it just in case.

**Comment by pagewin** [comment]
Dependent on #170 - revisit and see if it's really a problem
